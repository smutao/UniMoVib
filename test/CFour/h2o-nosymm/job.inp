H2O without symmetry

 $contrl
   qcprog="cfour"
 $end

 $qcdata
   fchk="h2o-c1.out"
   geom="GRD"
 $end
