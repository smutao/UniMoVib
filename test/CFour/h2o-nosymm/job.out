 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.0.0, MAY 02, 2017
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      CFOUR
 FCHK file:       h2o-c1.out
 GEOM file:       GRD


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    H       1            0.00000000   -0.75387902    0.52702010            1.00782503
     2    O       8            0.00000000    0.00000000   -0.06641412           15.99491463
     3    H       1            0.00000000    0.75387902    0.52702010            1.00782503
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   C2v 
 Molecular Point Group (Z+M) :   C2v 


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU),
 force constants (mDyn/A), IR intensities (km/mol)

                                                   1                                 2                                 3
          Irreps                                  A1                                A1                                B2
     Frequencies                           1668.8829                         3840.9218                         3945.5306
  Reduced masses                              1.0826                            1.0453                            1.0809
 Force constants                              1.7765                            9.0855                            9.9143
  IR intensities                             62.9912                            3.2140                           39.0254
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1         0.00000  -0.42835  -0.56038       0.00000   0.58429  -0.39665       0.00000  -0.55428   0.43631
           2   8         0.00000  -0.00000   0.07062       0.00000   0.00000   0.04999       0.00000   0.06985  -0.00000
           3   1         0.00000   0.42835  -0.56038       0.00000  -0.58432  -0.39667       0.00000  -0.55424  -0.43629

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                  B1                                B2                                A1
     Frequencies                              0.0000                           -0.0000                           -1.2113
  Reduced masses                              6.0035                            6.0035                            6.0035
 Force constants                              0.0000                           -0.0000                           -0.0000
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735
           2   8         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735
           3   1         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735

                                                   4                                 5                                 6
          Irreps                                  B1                                A2                                B2
     Frequencies                              0.0000                            0.0000                            3.6768
  Reduced masses                              1.1259                            1.0078                            1.0468
 Force constants                              0.0000                            0.0000                            0.0000
  IR intensities                            245.8245                            0.0000                           87.2583
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1         0.70432   0.00000   0.00000       0.70711   0.00000   0.00000       0.00000  -0.40461  -0.57878
           2   8        -0.08876  -0.00000  -0.00000       0.00000   0.00000   0.00000       0.00000   0.05099   0.00000
           3   1         0.70432   0.00000   0.00000      -0.70711   0.00000  -0.00000       0.00000  -0.40461   0.57878


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Temperature               :       298.150000    Kelvin
 Pressure                  :         1.000000    Atm
 Molecular mass            :        18.010565    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :         2
 The C2v  point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                  2.251194            4.090878            6.342072
           X                         0.000000            0.000000            1.000000
           Y                         1.000000            0.000000            0.000000
           Z                         0.000000            1.000000            0.000000

 Rotational temperatures            38.474605           21.172427           13.657021    Kelvin
 Rot. constants A, B, C             26.741222           14.715591            9.492116    cm**-1
                                   801.681656          441.162315          284.566481    GHz

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.021541           13.517079
 Thermal correction to Energy               :            0.024376           15.296053
 Thermal correction to Enthalpy             :            0.025320           15.888539
 Thermal correction to Gibbs Free Energy    :            0.003901            2.447661


 Job terminated correctly! Tue May 02 17:55:02 2017
